azureml/eval_prompts.yml

```diff
@@ -0,0 +1,32 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandJob.schema.json
+command: >
+  python -m autora.doc.pipelines.main eval-prompts
+  ${{inputs.data_dir}}/data.jsonl
+  ${{inputs.data_dir}}/all_prompt.json
```

Comment:
Let's make the prompts file its own input parameter. This will make it easier to run experiments.

Reply:
should I parameterize the data file as well? The default value can be 'data.jsonl' but a user can modify input if they want. WDYT?